Gemfile

```diff
@@ -188,7 +189,7 @@ gem 'lograge'
 gem 'rack-ssl-enforcer'
 
 # PubSub for NetSim
-gem 'pusher', '~> 0.14.5'
+gem 'pusher', '~> 1.0'
```

Comment:
no, I've been breaking out single-gem upgrades where possible. Let's consider #10281 a prerequisite to this PR and merge it in before this one.